drivers/kscan/kscan_ite_it8xxx2.c

```diff
@@ -19,6 +21,8 @@
 #define LOG_LEVEL CONFIG_KSCAN_LOG_LEVEL
 LOG_MODULE_REGISTER(kscan_ite_it8xxx2);
 
+#define KEYBOARD_KSI_PIN_COUNT		0x8
+#define KEYBOARD_KSI_PIN_MASK		0xff
```

Comment:
This macro assumes that all KSI pins are connected to the same wake up controller.  I'm not sure if it's possible to create build assert to verify this.  You could add a run-time to check to `kscan_it8xxx2_init()`.

You should also set the KS pin mask value from the devicetree.  You can do this at runtime as shown below, or you can probably use LISTIFY() to create the mask at compile time.

```
const struct device *kscan_wucs = config->wuc_map_list[0].wucs;
config->ksi_pin_mask = 0;
for (int i = 0; i < KEYBOARD_KSI_PIN_COUNT; i++) {
    if (IS_ENABLED(CONFIG_LOG)) {
        if (config->wuc_map_list[i].wucs != kscan_wucs) {
            LOG_ERR(...);
        }
    }
    config->ksi_pin_mask |= config->wuc_map_list[i].mask;

```

Reply:
Done, but I move ksi_pin_mask to data struct, since it's not a property.